cherrypy/lib/cpstats.py

```diff
@@ -308,6 +308,13 @@ def _get_threading_ident():
     return threading._get_ident()
 
 
+def _get_response_status(resp):
+    response_status = getattr(resp, 'output_status', None) or resp.status
+    if sys.version_info > (3,):
```

Comment:
we use `six.PY3` for this.

Reply:
But it's not needed, because it can be replaced by the function, which encapsulates this check better.